keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1470,6 +1490,13 @@ type ResourceClaimSpec struct {
     // +optional
     ResourceClassName string
 
+    // A claim with ManagementAccess set to true gets allocated without
+    // marking the underlying resource as in-use. This is a privileged
+    // access mode that is only granted when there is a Quota object
+    // in the same namespace as the claim where AllowManagementAccess
+    // is true.
+    ManagementAccess bool
```

Comment:
This also needs to be in the allocation result, together with some explanations that the DRA driver kubelet plugin should validate allocation results. It's always possible that an error occurred and two claims both have the same instance in their result. The plugin should catch that and reject preparing the second claim, except when management access is involved.

Reply:
This is also necessary because devices might be injected differently if they are meant for management access vs. normal allocation (this is true for NVIDIA GPUs with MIG devices, for example).

Reply:
I added a comment saying that the `ResourceClaimSpec.ManagementAccess` is immutable. Then this information will be available to drivers - assuming that we do https://github.com/kubernetes/enhancements/pull/4615.